File: my-folder/0215-kth-largest-element-in-an-array/solution.java
class Solution {
    public int findKthLargest(int[] nums, int k) {
        PriorityQueue<Integer> p = new PriorityQueue<>(Collections.reverseOrder());

        for(int i : nums){
            p.offer(i);
        }

        for(int i = 1; i < k; i++){
            p.poll();
        }
        return p.poll();


    }
}

File: my-folder/0215-kth-largest-element-in-an-array/solution.py
class Solution:
    def findKthLargest(self, nums: List[int], k: int) -> int:
        freq = {}

        for num in nums:
            if num not in freq:
                freq[num] = 0
            freq[num] += 1
        
        larg = 0
        for i in range(10000, -10000, -1):
            if i not in freq: continue
            larg += freq[i]
            if larg >= k:
                return i
        

        return -67676767676776
